apps/opik-backend/src/main/java/com/comet/opik/domain/DatasetItemSummary.java

```diff
@@ -0,0 +1,9 @@
+package com.comet.opik.domain;
+
+import java.util.UUID;
+
+public record DatasetItemSummary(UUID datasetId, long datasetItemsCount) {
+    public static DatasetItemSummary empty(UUID datasetId) {
```

Comment:
Very minor: annotate arguments with Lombok non-null where it makes sense.

Reply:
Done